en/custom_build/GitBranching.md

```diff
@@ -0,0 +1,55 @@
+# Git Branching Strategy for Custom Builds
```

Comment:
I have subedited, primarily for my taste. Please check it again, but this is pretty much the same content.